script/foreman-debug

```diff
@@ -290,7 +305,8 @@ if [ $NOGENERIC -eq 0 ]; then
   add_cmd "ausearch -m AVC -m USER_AVC -m SELINUX_ERR | head -n 100" "selinux_first_denials.log"
   add_cmd "ausearch -m AVC -m USER_AVC -m SELINUX_ERR || grep AVC /var/log/audit/audit.log" "selinux_denials.log"
   if [ -f /usr/sbin/selinuxenabled ] && /usr/sbin/selinuxenabled; then
-    add_cmd "sepolgen-ifgen &>/dev/null && audit2allow -Ra || audit2allow -a" "selinux_audit2allow"
+    [[ -f /var/lib/sepolgen/interface_info ]] || sepolgen-ifgen &>/dev/null
```

Comment:
I would recommend keeping this change in separate issue

Reply:
There's no time, removed.